worlds/ffmq/Regions.py

```diff
@@ -211,21 +211,18 @@ def stage_set_rules(multiworld):
     # If there's no enemies, there's no repeatable income sources
     no_enemies_players = [player for player in multiworld.get_game_players("Final Fantasy Mystic Quest")
                           if multiworld.worlds[player].options.enemies_density == "none"]
-    if (len([item for item in multiworld.itempool if item.classification in (ItemClassification.filler,
-            ItemClassification.trap)]) > len([player for player in no_enemies_players if
-                                              multiworld.worlds[player].options.accessibility == "minimal"]) * 3):
+    if (len([item for item in multiworld.itempool if item.excludable
+            ]) > len([player for player in no_enemies_players if
+                                              multiworld.worlds[player].options.accessibility != "minimal"]) * 3):
```

Comment:
Those are some interesting indents and line breaks...

I think this is how I would do it:
```suggestion
    if (
        len([item for item in multiworld.itempool if item.excludable]) >
        len([player
             for player in no_enemies_players
             if multiworld.worlds[player].options.accessibility != "minimal"]) * 3
    ):
```

Reply:
I like this more, I just kept the old formatting

Reply:
You could notice that I sneaked in a disagreement that I have with PEP-8 here. :)

<details><summary>...</summary>Line breaks go after binary operators! Fight me!!</details>